pkgs/development/python-modules/segno/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, sphinx
+, sphinx-paramlinks
+, sphinx_rtd_theme
+, pytestCheckHook
+, pypng
+, pyzbar
+}:
+
+buildPythonPackage rec {
+  pname = "segno";
+  version = "1.5.2";
+
+  src = fetchFromGitHub {
+    owner = "heuer";
+    repo = "segno";
+    rev = version;
+    sha256 = "sha256-+OEXG5OvrZ5Ft7IO/7zodf+SgiRF+frwjltrBENNnHo=";
+  };
+
+  propagatedBuildInputs = [
+    sphinx
+    sphinx-paramlinks
+    sphinx_rtd_theme
```

Comment:
We only require those if we are building the docs. I am not sure if we do that but for that the should be in nativeBuildInputs.

Reply:
Ah good catch, yes! I've removed them - and sphix-paramlinks as it isn't used anymore.